src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -504,3 +504,34 @@ induceEAM p m = es m `AM.overlay` vs m
   where
     es = AM.edges . filter p . AM.edgeList
     vs = AM.vertices . AM.vertexList
+
+-- | Constructs an acyclic graph from any graph based on
+-- the order of vertices to produce an acyclic graph.
+-- The internal order defines the valid set of edges.
+-- 
+-- The order for valid edges is \<, ie. for any two
+-- vertices x and y (x \> y), the only possible edge is (y, x).
+-- This will guarantee the production of an acyclic graph since
+-- no back edges are possible.
+--
+-- Topological orderings are also closely related to the concept
+-- of a linear extension of a partial order in mathematics.
+-- Please look at <https://en.wikipedia.org/wiki/Topological_sorting#Relation_to_partial_orders Relation to partial orders> for
+-- additional information. 
+-- In this case the partial order is the order of the vertices
+-- itself. And hence, the topological ordering for such graphs
+-- is simply its `vertexList`.
```

Comment:
Please be careful with Haddock: I believe this markup syntax is not supported. Double-check the generated documentation.

Reply:
Ah, my bad. I did check for the most part. I'll go through it again, in the meanwhile please let me know if you've found anything else.